src/Analysis/pipeline/action.py

```diff
@@ -0,0 +1,9 @@
+class Action():
```

Comment:
K cemu je tato trida? Vypada to jako predek akci, ale nedefinuje to zadne spolecne rozhrani.